metadata.json

```diff
@@ -9,10 +9,6 @@
   "issues_url": "https://github.com/autostructure/artifactory/issues",
   "dependencies": [
     {
-      "name": "maestrodev-wget",
```

Comment:
Since moving to downloading connectors using a `file` resource, this module isn't needed anymore.